__mocks__/@inrupt/solid-client.js

```diff
@@ -0,0 +1,15 @@
+import { vi } from 'vitest'
+import { mockSolidDatasetFrom } from '@inrupt/solid-client';
+
+export * from '@inrupt/solid-client';
```

Comment:
Vitest overwrites the module export with whatever we have in this file. So we want to re-export any functions we're not mocking.